### PR TITLE
Update Tuya temperature sensor variants

### PR DESCRIPTION
**Pull request by maxustm**
replace "_TZE200_utkemkbs" to var4 and add new one "_TZE204_utkemkbs" (Roximo's T&H sensors)

### CODE REVIEW

File: zhaquirks/tuya/ts0601_sensor.py [hunk 1]
```diff
@@ -251,6 +250,8 @@ class TuyaTempHumiditySensorVar04(CustomDevice):
             ("_TZE200_yjjdcqsq", "TS0601"),
             ("_TZE200_9yapgbuv", "TS0601"),
             ("_TZE204_yjjdcqsq", "TS0601"),
+            ("_TZE200_utkemkbs", "TS0601"),
+            ("_TZE204_utkemkbs", "TS0601"),
```
Comment (TheJulianJES) [inline zhaquirks/tuya/ts0601_sensor.py:254]: ~~These lines are duplicated now.~~
EDIT: Nvm

**Review by TheJulianJES — COMMENTED** [review]
Thanks for the PR!
Are the changes tested and working for both sensors?
Also, why did the `_TZE200_utkemkbs` need to move from `Var03` to `Var04`?

**Comment by maxustm** [comment]
> Thanks for the PR! Are the changes tested and working for both sensors? Also, why did the `_TZE200_utkemkbs` need to move from `Var03` to `Var04`?

Move to var4 - to support battery information.
Both sensors have been tested and work normal.

**Review by TheJulianJES — APPROVED** [review]
Thanks!

**Pull request merged**